integration_tests/pages/refer/show/additionalInformation.ts

```diff
@@ -0,0 +1,28 @@
+import { CourseUtils, DateUtils } from '../../../../server/utils'
+import Page from '../../page'
+import type { Course, Referral } from '@accredited-programmes/models'
+
+export default class AdditionalInformationPage extends Page {
+  referral: Referral
+
+  constructor(args: { course: Course; referral: Referral }) {
+    const { course, referral } = args
+
+    const coursePresenter = CourseUtils.presentCourse(course)
+
+    super(`Referral to ${coursePresenter.nameAndAlternateName}`)
+
+    this.referral = referral
+  }
+
+  shouldContainAdditionalInformation() {
+    cy.get('[data-testid="submitted-text"]').should(
+      'contain.text',
+      `Submitted in referral on ${DateUtils.govukFormattedFullDateString(this.referral.submittedOn)}.`,
+    )
+  }
+
+  shouldContainSubmittedText() {
+    cy.get('[data-testid="additional-information"]').should('have.text', this.referral.additionalInformation)
```

Comment:
probably I'm not understand how/where the `referral` test fixture is defined but I can't see here or in the cypress test how we're verifying that `this.referral.additionaInformation` isn't null/empty/undefined...

Reply:
Not entirely related, but it looks like the names of the two `should` methods are the wrong way round here

Reply:
@edavey Good point. We pass a Referral to this page class on instantiation, and it's a required field on a Referral so I think this is ok for now. I've spotted a couple of things in our tests that could probably be made more robust, which we can address in a future task. I'll add a card to our backlog for this.

@yndajas good spot! Must have been my end of the day brain. Have fixed this.